twitter-borrowbot/twitterbot.py

```diff
@@ -1,21 +1,21 @@
 #-*- encoding: utf-8 -*-
 import os
-import re
 import time
-import requests
 import tweepy
-
-from services import InternetArchive, ISBNFinder
 from dotenv import load_dotenv
 
+from services import InternetArchive, ISBNFinder, Logger
+
 ACTIONS = ('read', 'borrow', 'preview')
 READ_OPTIONS = dict(zip(InternetArchive.MODES, ACTIONS))
 BOT_NAME = "@borrowbot"
 STATE_FILE = 'last_seen_id.txt'
 
+LOGGER = Logger("./logs/tweet_logs.txt", "./logs/error_logs.txt")
```

Comment:
We probably don't want to roll our own Logger because, among other reasons, what we're doing is likely not thread safe ;)